drone/server/server.go

```diff
@@ -415,7 +416,7 @@ func setupEvilGlobals(c *cli.Context, v store.Store) {
 	droneserver.Config.Server.Cert = c.String("server-cert")
 	droneserver.Config.Server.Key = c.String("server-key")
 	droneserver.Config.Server.Pass = c.String("agent-secret")
-	droneserver.Config.Server.Host = c.String("server-host")
+	droneserver.Config.Server.Host = strings.TrimRight(c.String("server-host"), "/")
```

Comment:
@bradrydzewski `TrimSuffix` can't resolve this issue if `DRONE_HOST` was set as `http://example.com/////`